horace_core/sqw/@sqw/private/init_sqw_from_file_.m

```diff
@@ -0,0 +1,35 @@
+function obj = init_sqw_from_file_(obj, in_struc)
```

Comment:
Is this related to masking?

Reply:
it is related to the `test_mask` as the test_mask behaves differently depending on the size of the mem_chunk, testing either filebacked or memory backed pixels as function of memory chunk. The change force test to validate specific part of the code